### PR TITLE
Update Basque language

### PR DESCRIPTION
**Pull request by OSFr0g**
## What does this PR do?
Adds many new lines to the basque language file to catch up with the latest robosats updates.
Deleted 3 lines that were previously removed in the english file.

## Checklist before merging
- [x] I have ran prettier `cd frontend; npm run format`.

### CODE REVIEW

**Comment by Reckless-Satoshi** [comment]
Awesome! Thank you for keeping Basque up to date :rocket:

File: frontend/static/locales/eu.json [hunk 1]
```diff
@@ -114,6 +116,53 @@
   "No orders found to buy BTC for {{currencyCode}}": "Eskaerarik ez BTC {{currencyCode}}gatik erosteko",
   "Filter has no results": "Iragazkiak ez du emaitzarik",
   "Be the first one to create an order": "Lehena izan eskaera bat sortzen",
+  "Orders per page:": "Orrialde bakoitzeko eskaerak:",
```
Comment (4pt1x) [inline frontend/static/locales/eu.json:119]: Could be shortened by: "Eskaerak orrialdeko:"

File: frontend/static/locales/eu.json [hunk 2]
```diff
@@ -114,6 +116,53 @@
...
+  "No rows": "Filarik ez",
```
Comment (4pt1x) [inline frontend/static/locales/eu.json:120]: I propose: "Lerrorik ez"

File: frontend/static/locales/eu.json [hunk 3]
```diff
@@ -397,6 +463,31 @@
   "Your TXID": "Zure TXID",
   "Lightning": "Lightning",
   "Onchain": "Onchain",
+  "open_dispute": "Gatazka bat irekitzeko itxaron behar duzu <1><1/>",
```
Comment (4pt1x) [inline frontend/static/locales/eu.json:466]: I don't clearly know the context, but the literal translation would be: "Ireki gatazka"

**Review by 4pt1x — COMMENTED** [review]
3 comments/suggestions added.

**Pull request merged**